examples/window/window_settings.rs

```diff
@@ -93,6 +95,20 @@ fn toggle_cursor(mut windows: Query<&mut Window>, input: Res<Input<KeyCode>>) {
     }
 }
 
+// This system will toggle the color theme used by the window
+fn toggle_theme(mut windows: Query<&mut Window>, input: Res<Input<KeyCode>>) {
+    if input.just_pressed(KeyCode::F) {
+        let mut window = windows.single_mut();
+
+        if let Some(current_theme) = window.window_theme() {
```

Comment:
Just to clarify, in unsupported platforms this will always return `None`, right?

Reply:
Yep, tested in Wayland, returns None.